core/src/main/java/com/csun/game/screens/TitleScreen.java

```diff
@@ -64,6 +64,7 @@ public void show() {
         assetManager.load("doctor.atlas", TextureAtlas.class);
         assetManager.load("mom.atlas", TextureAtlas.class);
         assetManager.load("skeleton.atlas", TextureAtlas.class);
+        assetManager.load("atlas/ui/ui.atlas", TextureAtlas.class);
```

Comment:
I see that you're loading atlas/ui/ui.atlas here but calling 

 TextureAtlas uiTextureAtlas = new TextureAtlas(Gdx.files.internal("atlas/ui/ui.atlas"));

in PauseSettingsTable line 32